### PR TITLE
Add use-gl argument to puppeteer

### PR DESCRIPTION
**Pull request by arthurmchr**
Enabling hardware acceleration with this option makes previews more consistent when using webgl.
As noted on puppeteer git, "Chrome/Chromium requires --use-gl=egl to enable GPU acceleration in headless mode."
Link to doc : https://peter.sh/experiments/chromium-command-line-switches/#use-gl

### CODE REVIEW

**Comment by ciphrd** [comment]
Thanks for the PR, I need to make some tests to check if this works as intended because we noticed weird behaviors in our GPU compute environments

**Comment by ciphrd** [comment]
When adding this flag, the `chrome://gpu` page outputs the following data:

image

However by removing the `--use-angle=gl-egl` flag, effectively replacing it by this one, we do get the GPU acceleration back:

image

I did some comparative tests using the previous docker image on a GPU intensive project, with the same capture settings:

GPU-current-300s-ooXcncyeb84mQ7htgfz51CfTmgsCio2393UnWdUfvN3AtfZ2kLZ
_--use-angle=gl-egl_

GPU-improved-300s-ooXcncyeb84mQ7htgfz51CfTmgsCio2393UnWdUfvN3AtfZ2kLZ
_--use-gl=egl_

Basically, this demonstrates that with this new flag, the GPU instances are properly utilizing the GPUs because they can now render way more frames than what they used to do. I suspect they were previously not properly using GPUs.

I'm going to reflect the changes in your PR and push the changes to prod.

I think the GPU community is going to be pretty happy about this :D

**Comment by arthurmchr** [comment]
Thanks for the fast research, looks great !
